gradle.properties

```diff
@@ -25,7 +25,7 @@ gradleMkdocsPluginVersion=1.1.0
 
 # Version and base tags can be overridden at build time.
 connectorVersion=0.4.0-SNAPSHOT
-pravegaVersion=0.4.0-50.da91e55-SNAPSHOT
+pravegaVersion=0.5.0-49.b80b627-SNAPSHOT
```

Comment:
I'm confused, why are we moving to `master`? Keep in mind that the API refactoring was merged to `master` (`0.5.0-SNAPSHOT`), not `r0.4`.

Reply:
I believe the issue is noticed with the connector/master code tested against the latest Pravega/master.

Reply:
Ok, to confirm, we want to move flink connector master to pravega master? we might want to create an `r0.4` branch first, no?